apps/storefront/src/utils/b3checkout.ts

```diff
@@ -28,8 +28,9 @@ export const attemptCheckoutLoginAndRedirect = async (
   }
 };
 
-export const setQuoteToStorage = (quoteId: string, date: any) => {
+export const setQuoteToStorage = (quoteId: string, date: any, quoteUuid: string | undefined) => {
```

Comment:
🤔 looking at the `searchParams.get('uuid')`, I see we are defaulting to `null`, but here it is `undefined`...